cumulusci/cli/robot.py

```diff
@@ -0,0 +1,107 @@
+import sys
+
+import click
+import pkg_resources
+import sarge
+
+from .runtime import pass_runtime
+
+
+@click.group("robot", help="Commands for working with robot framework")
+def robot():
+    pass
+
+
+# Commands for group: robot
+
+
+@robot.command(
+    name="install_playwright",
+    help="Install libraries necessary to use playwright with robot framework",
+)
+@pass_runtime(require_project=False, require_keychain=False)  # maybe not needed...
+@click.option("-n", "--dry_run", is_flag=True)
+def robot_install_playwright(runtime, dry_run):
+    _require_npm()
+    if _is_package_installed("robotframework-browser"):
+        click.echo("Playwright support seems to already have been installed")
+    else:
+        _install_browser_library(dry_run)
+        _initialize_browser_library(dry_run)
+
+
+@robot.command(
+    name="uninstall_playwright",
+    help="uninstalls the robotframework-browser package and node modules",
+)
+def robot_uninstall_playwright():
+    """Attempt to uninstall playwright"""
+    p1 = sarge.Command([sys.executable, "-m", "Browser.entry", "clean-node"])
+    p2 = sarge.Command(
+        [sys.executable, "-m", "pip", "uninstall", "robotframework-browser", "--yes"]
+    )
+
+    click.echo("removing node modules...")
+    click.echo(f"running {' '.join(p1.args)}")
+    p1.run()
+    click.echo("removing python module robotframework-browser...")
+    click.echo(f"running {' '.join(p2.args)}")
+    p2.run()
```

Comment:
❓: Do we want to add a check of the return code here? 

My thinking is that the user would expect the command to tell them whether uninstallation was successful.

Reply:
I added a "uninstallation complete" message